content/hi/DevSecOps.md

```diff
@@ -0,0 +1,34 @@
+| Title                                    | Status    | Category | Exclude_Search |
+| ---------------------------------------- | --------- | -------- | -------------- |
+| [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) ( डेवॉप्स) | Completed | अवधारणा  | True           |
+
+#यह क्या है
+
+सॉफ्टवेयर विकास जीवनचक्र के हर चरण में [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) स्वचालित रूप से सुरक्षा में साथ निभाना है | जिससे Agile और [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) की गति से सुरक्षित सॉफ़्टवेयर का विकास संभव हो जाता है।
+
+#समस्या
+
+आज की डिजिटल दुनिया में आईटी सुरक्षा एक महत्वपूर्ण मुद्दा है, और खतरे रातोंरात दूर नहीं होंगे। साइबर हमले और धोखाधड़ी बढ़ रही है। इस कठोर वास्तविकता का सामना करते हुए, यह समझ से बाहर है कि आज कोई भी संगठन DevSecOps कार्यप्रणाली के सुरक्षा पहलू की उपेक्षा करेगा।
+[DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) सुरक्षा की चुनौतियों को संक्षेप में प्रस्तुत करने के लिए:
+
+1. [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) टीमें सुरक्षा को एक उपद्रव के रूप में देखती हैं
+
+2. IT सुरक्षा दल [DevOps](https://github.com/cncf/glossary/blob/main/content/hi/devops.md) की तेज़ गति के साथ तालमेल नहीं बिठा पा रहे हैं
+
+3. कई ओपन-सोर्स और अपरिपक्व टूल में अपर्याप्त सुरक्षा सुविधाएं होती हैं
+
+4. अपर्याप्त रूप से प्रबंधित विशेषाधिकार प्राप्त अभिगम नियंत्रण के परिणामस्वरूप हमले के अधिक अवसर मिलते हैं
+
+   #समाधान
+
+DevSecOps  के लाभों को ध्यान में रखते हुए, इसे अभी भी व्यापक रूप से लागू नहीं किया जा रहा है। कम से कम, अभी तक। आइए DevSecOps  को अपनाने के लाभों के बारे में गहराई से जानें:
```

Comment:
@SahityaGupta25 This whole section doesn't seem like the correct localisation from the English version. Please use https://github.com/cncf/glossary/blob/main/content/en/devsecops.md as reference.